lib/mobility-core/src/Kernel/Beam/Lib/UtilsTH.hs

```diff
@@ -369,17 +372,18 @@ mkTableInstances' name table mbSchema tableFieldModifier = do
   serialInstances <- mkSerialInstances name
   fromJSONInstances <- mkFromJSONInstance name
   toJSONInstances <- mkToJSONInstance name
-  customMappings <- mkCustomMappings name tableFieldModifier
+  customMappings <- mkCustomMappings name table tableFieldModifier
```

Comment:
not sure about this, because model was in camelCase, now in snakeCase
see line 101
`let tableNameD = FunD 'KV.tableName [Clause [] (NormalB (LitE (StringL $ init $ camel (nameBase name)))) []]`

Reply:
we are using for drainer and euler we are doing pascal so it should be fine